cppcache/integration-test/CMakeLists.txt

```diff
@@ -81,6 +81,12 @@ foreach(FILE ${SOURCES})
   add_executable(${TEST} ${FILE})
   set_target_properties(${TEST} PROPERTIES FOLDER test/cppTests)
 
+  if(${CMAKE_CXX_COMPILER_ID} MATCHES "GNU" AND ${USE_CPP_COVERAGE})
+      set_target_properties(${TEST}
+          PROPERTIES APPEND_STRING COMPILE_FLAGS "--coverage"
+                     APPEND_STRING LINK_FLAGS "--coverage")
```

Comment:
Are we sure that integration tests have to be compiled with coverage? Would it be sufficient to just link?

Reply:
The lcov docs indicate that you need both:
http://ltp.sourceforge.net/coverage/lcov.php